src/TorchSharp/TorchVision/Ops/Misc.cs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) .NET Foundation and Contributors.  All Rights Reserved.  See LICENSE in the project root for license information.
+
+// A number of implementation details in this file have been translated from the Python version of torchvision,
+// largely located in the files found in this folder:
+//
+// https://github.com/pytorch/vision/blob/a4f53308b2d0f1aa9191686e326f45c26053f686/torchvision/ops/misc.py
+//
+// The origin has the following copyright notice and license:
+//
+// https://github.com/pytorch/audio/blob/main/LICENSE
+//
+
+using System;
+using System.Collections.Generic;
+using static TorchSharp.torch;
+
+#nullable enable
+namespace TorchSharp
+{
+    public static partial class torchvision
+    {
+        public static partial class ops
+        {
+            /// <summary>
+            /// Configurable block used for Convolution2d-Normalization-Activation blocks.
+            /// </summary>
+            /// <param name="in_channels">Number of channels in the input image</param>
+            /// <param name="out_channels">Number of channels produced by the Convolution-Normalization-Activation block</param>
+            /// <param name="kernel_size">Size of the convolving kernel.</param>
+            /// <param name="stride">Stride of the convolution.</param>
+            /// <param name="padding">Padding added to all four sides of the input. Default: null, in which case it will calculated as ``padding = (kernel_size - 1) // 2 * dilation``</param>
+            /// <param name="groups">Number of blocked connections from input channels to output channels.</param>
+            /// <param name="norm_layer">Norm layer that will be stacked on top of the convolution layer. If ``null``, ``torch.nn.BatchNorm2d`` is used.</param>
+            /// <param name="activation_layer">Activation function which will be stacked on top of the normalization layer (if not None), otherwise on top of the conv layer. If ``null``, ``torch.nn.ReLU`` is used.</param>
+            /// <param name="dilation">Spacing between kernel elements.</param>
+            /// <param name="inplace">Parameter for the activation layer, which can optionally do the operation in-place.</param>
+            /// <param name="bias">Whether to use bias in the convolution layer. By default, is false.</param>
+            public static nn.Module Conv2dNormActivation(
```

Comment:
Wat about Conv1d?

Reply:
`Conv1dNormActivation` is not included in the original `torchvision`